src/cephadm/cephadm

```diff
@@ -112,6 +112,200 @@ class Ceph(object):
 
 ##################################
 
+class Firewalld(object):
```

Comment:
Let's make this an abstract class instead. I'm guessing that someone will eventually use a different firewall. 
Using an abstract implementation makes this easier.

Reply:
really? sounds a bit overkill to me.

Reply:
I think that maybe we should be centered in this moment to in just making it work, and to deal successfully with all the firewall functionality use cases before to start introducing other kind of firewalls.